src/ape/utils.py

```diff
@@ -195,7 +185,6 @@ def extract_nested_value(root: Mapping, *args: str) -> Optional[Dict]:
 
 
 __all__ = [
-    "cached_property",
```

Comment:
So I did sort of intend these to be exported for use by external libraries (2nd/3rd class plugins) because I generally thought them to be useful. Also this way in case we needed to patch any of their functionality to work with how ape's API classes work, I wanted plugins to source these utilities from us to make that patching easier.

Overall unsure of the pros/cons of this approach.

Reply:
Ah, I figured we were only doing that because of python 3.7.

It's like, if python 3.7 goes away, I'd like to just delete this code without worrying about making breaking changes, that is the main benefit. But if there is a chance of making custom implementations for these methods, than it does makes sense to keep them in `ape.utils.py`

Reply:
Yeah, so like I wouldn't modify them too much, but let's say there was a bug in the implementation that caused us problems, I'd want it to be easier to fix by modifying here instead of all downstream users that need the same fix